app/src/views/FieldReportForm/common.ts

```diff
@@ -443,6 +462,11 @@ export const reportSchema: FormSchema = {
                 // SITUATION - EVT
                 return {
                     ...baseSchemaTwo,
+                    situationalOverviewConsented: {
```

Comment:
We need to udpate the fieldsheet when we do these changes. @babinkarmacharya79 @Shhhhhubh